lib/advanced-open-file-view.coffee

```diff
@@ -64,7 +67,10 @@ class AdvancedFileView extends View
 
   @content: (params) ->
     @div class: 'advanced-open-file', =>
-      @p outlet:'message', class:'icon icon-file-add', "Enter the path for the file/directory. Directories end with a '" + path.sep + "'."
+      @p
+        outlet:'message',
+        class:'icon icon-file-add',
+        "Enter the path for the file/directory. Directories end with a '" + path.sep + "'."
```

Comment:
Use string interpolation here instead of the `+`; should be:

```coffee
"Enter the path for the file/directory. Directories end with a '#{path.sep}'."
```